tests/common/mock.rs

```diff
@@ -172,8 +172,16 @@ pub fn new_setup() -> Setup {
 			carl: keys[2].clone(),
 		},
 		fids: FIDs {
-			alice: Perun::calc_funding_id(cid, &params.participants[0]),
-			bob: Perun::calc_funding_id(cid, &params.participants[1]),
+			alice: FundingOf::<Test> {
```

Comment:
Please keep `calc_funding_id` and revert the change. It looked way nicer before. You can create the funding object within `calc_funding_id`.